src/ProjectTemplates/Web.ProjectTemplates/content/Worker-CSharp/Worker.cs

```diff
@@ -22,7 +22,14 @@ protected override async Task ExecuteAsync(CancellationToken stoppingToken)
             while (!stoppingToken.IsCancellationRequested)
             {
                 _logger.LogInformation("Worker running at: {time}", DateTimeOffset.Now);
-                await Task.Delay(1000, stoppingToken);
+                try
+                {
+                    await Task.Delay(1000, stoppingToken);
+                }
+                catch (TaskCanceledException)
```

Comment:
OperationCancelledException

Reply:
Any specific reason, besides it covering more potential exception types?

Reply:
More generic? I usually use it everywhere. TaskCanceledException only if you want a handle on the cancelled task.